test/committee/helpers_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Committee.HelpersTest do
+  use ExUnit.Case
+  import Committee.TestHelpers
+  alias Committee.Helpers
+
+  describe "staged_files/0" do
+    test "returns the staged files", context do
+      in_tmp(context.test, fn ->
```

Comment:
what does `context.test` return?

Reply:
The message of the test. In this case it is `returns the staged files`.